src/constants/collectionCoverImage.ts

```diff
@@ -0,0 +1,42 @@
+export const mainPageCollections = [
+  {
+    title: "Dresses",
+    pathName: "/items?category=dresses",
+    imageUrl: "/dresses/dress-cover.jpg",
```

Comment:
Something is wrong because images are broken.

<img width="1257" alt="image" src="https://github.com/OlhaPo/mariam-nextjs-typescript/assets/37477697/e5a7666b-139b-4a5f-8ec1-ae9f9f0b3860">